Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN pip install --no-cache-dir -r requirements.txt
 RUN apk del deps
 
 COPY . /usr/src/app
-RUN chmod +x /usr/src/app/contract_observer.py
+RUN chmod +x /usr/src/app/observers/contract_observer.py
```

Comment:
It doesn't look like anything calls `contract_observer.py` as an executable. We could remove this line.